pyaedt/modules/SolveSweeps.py

```diff
@@ -376,27 +348,27 @@ def __init__(self, setup, sweepname, sweeptype="Interpolating", save_fields=True
                 self.props = copy.deepcopy(Sweep3DLayout)
             # for t in props:
             #    _tuple2dict(t, self.props)
-            if SequenceMatcher(None, sweeptype.lower(), "interpolating").ratio() > 0.8:
-                sweeptype = "kInterpolating"
-            elif SequenceMatcher(None, sweeptype.lower(), "discrete").ratio() > 0.8:
-                sweeptype = "kDiscrete"
-            elif SequenceMatcher(None, sweeptype.lower(), "fast").ratio() > 0.8:
-                sweeptype = "kBroadbandFast"
+            if SequenceMatcher(None, sweep_type.lower(), "interpolating").ratio() > 0.8:
+                sweep_type = "kInterpolating"
+            elif SequenceMatcher(None, sweep_type.lower(), "discrete").ratio() > 0.8:
+                sweep_type = "kDiscrete"
+            elif SequenceMatcher(None, sweep_type.lower(), "fast").ratio() > 0.8:
+                sweep_type = "kBroadbandFast"
             else:
                 warnings.warn("Sweep type is invalid. `kInterpolating` is set as the default.")
-                sweeptype = "kInterpolating"
-            self.props["FreqSweepType"] = sweeptype
+                sweep_type = "kInterpolating"
+            self.props["FreqSweepType"] = sweep_type
             self.props["GenerateSurfaceCurrent"] = save_fields
 
     @property
     def combined_name(self):
-        """Compute the setupname : sweepname string.
+        """Compute the setup_name : sweep_name string.
```

Comment:
If this is a property, the description should be a noun: Something like:
"Combined setup and sweep names in a string in this format: ``setup_name : sweep_name``.